### PR TITLE
Added Autotweet editor panel for enable/disable Autotweet and color-coded icon in Pre-Publish panel

### LINKED ISSUE #176: Add AutoTweet panel (Slotfill) for enable/disable tweet on publish and color-coded icon in Pre-Publish panel title
### Is your enhancement related to a problem? Please describe.

Sub-task of #59 
**1. Add AutoTweet panel (Slotfill) for enable/disable tweet on publish and add color-coded icons Pre-Publish Panel title** 
- [x] Own standard slotfill, with default icon
- [ ] If Tweeting is disabled, show but disable controls (message text-area will be always visible under #178)
- [x] Label updates when enabled to say what it will do
- [x] Icon changes active state too, so that status can be seen (and read too) even if the slotfill is closed
- [x] State what the action will be, and use appropriate icon colour coding
- [ ] Message controls available in open state (message text-area will be always visible under #178)
- [x] Disabled state copy and icon updated

### Designs

DRAFT STATE
Default state:
image

Disabled state:
image

Enabled state:
image

PRE-PUBLISH CHECKS
Pre-publish - Enabled: 
Screen Shot 2019-12-18 at 10 32 22 PM

Pre-publish - Enabled & Open: 
Screen Shot 2019-12-18 at 10 32 53 PM

Pre-publish - Disabled: 
Screen Shot 2019-12-18 at 10 33 21 PM

### Describe alternatives you've considered

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by iamdharmesh**
### Description of the Change
As requested on #176, This PR adds slotfill for auto-tweet in the editor sidebar and also added a color-coded icon in the pre-publish panel. Please check screenshots for more information on this.

|Editor Sidebar ||
|---|---|
|Auto Tweet enabled |image|
|Auto Tweet Disabled |image|
|[Status] Tweeted Successfully|image|
|[Status] Tweet Failed|image|
|[Status] Not Tweeted|image|

|Pre-Publish Panel ||
|---|---|
|Auto Tweet enabled |image|
|Auto Tweet Disabled |image|

Closes #176 

### Possible Drawbacks
n/a

### Verification Process
1. Go To Add a new post.
2. Notice the dedicated panel for auto-tweet in the editor sidebar.
3. Test enable/disable tweet on publish
4. Click on publish
5. Notice the pre-publish panel for auto-tweet with icon
6. Test enable/disable tweet on publish
7. Publish the post
8. Verify the tweet status in panel of the editor sidebar (for failed, success, and not tweeted).

### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/10up/.github/blob/trunk/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### Credits

Props @iamdharmesh @jeffpaul @linawiezkowiak @oszkarnagy (for the UX/Design work)

### CODE REVIEW

File: assets/css/admin-autoshare-for-twitter.css [hunk 1]
```diff
@@ -142,3 +124,9 @@ tbody .autoshare-for-twitter-status-logo--enabled::before {
 tbody .autoshare-for-twitter-status-logo--disabled::before {
 	background-image: url('../images/twitter_disabled.svg');
 }
+
+.autoshare-for-twitter-pre-publish-panel svg.components-panel__icon,
+.autoshare-for-twitter-editor-panel svg.components-panel__icon{
+	width: 24px;
+	height: 24px;
+}
```
Comment (cadic) [inline assets/css/admin-autoshare-for-twitter.css]:
```suggestion
}

```

**Review by cadic — DISMISSED** [review]
LGTM!

**Comment by iamdharmesh** [comment]
Thanks for the review @cadic.

@jeffpaul can you please help to quick check on the wordings update in this PR? if anything needs to be updated.

Thanks

**Review by cadic — DISMISSED** [review]

File: src/js/index.js [hunk 2]
```diff
@@ -29,40 +36,37 @@ class AutoshareForTwitterPrePublishPanelPlugin extends Component {
 	maybeSetEnabledText() {
 		try {
 			const enabled = select( STORE ).getAutoshareEnabled();
-			const enabledText = enabled ? __( 'Enabled', 'autoshare-for-twitter' ) : __( 'Disabled', 'autoshare-for-twitter' );
+			const enabledText = enabled ? __( 'This post will be Tweeted', 'autoshare-for-twitter' ) : __( 'Will not be Tweeted', 'autoshare-for-twitter' );
```
Comment (jeffpaul) [inline src/js/index.js]: Perhaps change the negative option to `This post will not be Tweeted`?

**Comment by jeffpaul** [comment]
@iamdharmesh only one small copy suggestion, otherwise this looks great, thanks!

**Review by cadic — APPROVED** [review]

**Pull request merged**